posthog/migrations/0424_externaldatasource_sync_frequency.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.11 on 2024-06-06 15:43
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0421_set_all_survey_responses_to_be_strings"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="externaldatasource",
+            name="sync_frequency",
+            field=models.CharField(
+                blank=True,
+                choices=[("day", "Daily"), ("week", "Weekly"), ("month", "Monthly")],
+                default="day",
+                max_length=128,
+            ),
```

Comment:
I can imagine a world where this becomes something like a cron format in the DB, even though we only allow users to select day/weekly/monthly on the frontend. e.g. temporal schedule create allows different formats - https://docs.temporal.io/cli/schedule#create - this is likely one for later, but good to keep in mind